ideas/87-new-protocol.md

```diff
@@ -63,26 +63,54 @@ Reasons for that are following:
 - Contributor: @cammellos
 - PM: @chadyj
 
-### Requirements & Dependencies
+
+## Current status of the Swarm
+
+- Weekly sync call on Fridays (contact eric@status.im if you want to join)
+- Focus on solving issues, particularly this one https://github.com/status-im/status-react/issues/4034
+
+## Requirements & Dependencies
 * Describe the new protocol code in a clear and easy to understand way ("birdview" with just each distinct message type and what they represent in term of actions explained)
 * Describe each message type in detail, including all the information encapsulated in it (update whenever new features requiring protocol version inc are added)
 * Describe security mechanism of the new protocol, where the sym/asym keys are used, how they are updated/refreshed, how do we choose whisper topics, etc.
 * Deliver working code for testing, if we desire we can do that in 2 steps and first implement only contact exchange + 1-1 chats via new protocol, with old protocol code handling private + public group chats, and the second and final step will implement that + erase all obsolete old protocol code.
 
-### Minimum Viable Product
+## Minimum Viable Product
 Goal Date: 2018-04-10
 
-Description: The new protocol is merged and supports 1-1 and public group chats. Documentation is finalized in the document https://docs.google.com/document/d/1Qh2h07T_qepzEJ7IytmxwIdQAOsGHrvhXwZxuZtbwgc/edit
+### Description
+
+The new protocol is merged and supports 1-1 and public group chats. Documentation is finalized in the document https://docs.google.com/document/d/1Qh2h07T_qepzEJ7IytmxwIdQAOsGHrvhXwZxuZtbwgc/edit
+
+### Outcome 
+
+This goal has been successfully achieved at the goal date.
+
+### What we could improve
 
-### Iteration 1
+It would be valuable for external developpers and for the sake of tracking future evolutions to the protocol to have the documentation ported into a repository and generate a static website to expose the documentation to the outside world.
```

Comment:
We have a docs repo coming up, @alexm-status knows more #documentation

Reply:
looking forward to that. I think it can fit well in the context of the status-index swarm